### ISSUE TITLE
Parameter Viewer

### ISSUE THREAD

**Issue by AndersonRayner** [flyingk/kVIS3#123]
Both the ArduPilot and PX4 import scripts import the parameter values stored in the logs.  Maybe we could come up with a good way of displaying those?  Currently they are imported into the fds struct as a PARAM field

**Comment by flyingk** [comment]
Could be done with a custom (bsp) tab next to data viewer with a table view.

**Comment by AndersonRayner** [comment]
Good idea.  I think prototyping internally within the APM or PX4 bsp could be a good proof of concept, but might be worth thinking about introducing it into the fds file structure formally so all BSPs could use the functionality.

**Comment by flyingk** [comment]
Not sure if not too bsp specific… maybe give the ads field a bsp_ prefix for now.

Examples how to register a tab in bsp_id and the required ui definition are in uavmainframe bsp.
